terraform/alb/alb.tf

```diff
@@ -0,0 +1,30 @@
+resource "aws_lb" "lb" {
+  name               = "${var.projectname}-lb"
+  internal           = false
+  load_balancer_type = "application"
+  security_groups    = [aws_security_group.ecs-ec2-sg.id]
```

Comment:
where this aws_security_group.ecs-ec2-sg.id is coming from?

Reply:
from the resource Heaven created on ECS code